packages/react/package.json

```diff
@@ -5,8 +5,14 @@
   "main": "./dist/index.cjs.js",
   "module": "./dist/index.esm.js",
   "exports": {
-    "require": "./dist/index.cjs.js",
-    "default": "./dist/index.esm.js"
+    ".": {
+      "require": "./dist/index.cjs.js",
+      "default": "./dist/index.esm.js"
+    },
+    "./dist/index.css": {
```

Comment:
How about changing `./dist/index.css` to `./index.css`?

Reply:
That's a great idea. Let's consider it before the release of v4.